api/openapi-spec/swagger.json

```diff
@@ -2799,6 +2786,52 @@
         }
       }
     },
+    "io.argoproj.workflow.v1alpha1.Mutex": {
+      "description": "Mutex holds Mutex configuration",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "name of the mutex",
+          "type": "string"
+        }
+      }
+    },
+    "io.argoproj.workflow.v1alpha1.MutexHolding": {
+      "description": "MutexHolding holds the resource that holds the mutex lock.",
```

Comment:
maybe `Holds a reference to the resource which is stores who is currently holding the mutex` - or am I wrong?